src/snowcli/app/main_typer.py

```diff
@@ -1,6 +1,8 @@
+import sys
+
 import click
-from snowcli.cli.common.cli_global_context import cli_context
-from snowcli.cli.common.flags import DEFAULT_CONTEXT_SETTINGS
+from snowcli.cli.common.cli_global_context import cli_context, cli_context_manager
```

Comment:
```suggestion
from snowcli.cli.common.cli_global_context import cli_context
```

manager seems to be unused

Reply:
Done - a trash left after debug :/